RepetitionAndConcatenation.py

```diff
@@ -0,0 +1,9 @@
+def rep_cat(x, y):
+    x = str(x) * 10
+    y = str(y) * 5
+    result = x + y
+    print("The concatenation result between x & y is: " + result)
+    return result
+
+
+rep_cat(3, 5)
```

Comment:
Be more general. Input data from keyboard and add some validations ( be sure the input data are numbers,...).

Reply:
Done